app/controllers/bookings_controller.rb

```diff
@@ -1,8 +1,9 @@
-# Controller that implements the logic for the bookings of the articles. Articles are booked 
+# Controller that implements the logic for the bookings of the articles. Articles are booked
 # for a particular time frame and charged by a day rate.
 class BookingsController < ApplicationController
   def index
     @bookings = Booking.all
+    @user = current_user
   end
 
```

Comment:
The booking price is set as a parameter when we create the booking (as it is a required attribute). We actually calculate it on the booking form (we need to so that a user can see how the price changes when selecting dates), and pass it as a param